### DISCUSSION TITLE
Regarding compiling source code

### DISCUSSION THREAD

**Discussion by daisukidaisy** [Leather128/LeathersFunkinEngine#66] [Q&A]
I tried what was the most recent build yesterday, compiled it as HTML5 and it had several issues:
1. If you defocus the browser window and come back, the audio stops working entirely.
2. Entering freeplay causes (I think) a crash.

What causes these issues?

I tried to compile it as an .exe but unfortunately it had already taken over 2 hours before I had to stop the process for unrelated reasons. Does it normally take several hours to compile? How can I speed up the process or at least know whether it's still working or not, or how much progress has been made towards full compilation?

**Comment by Leather128** [comment]
well compiling to html5 is a very new and buggy thing since i was developing leather engine on windows for the most part so bugs are 100% expected. but it taking hours to compile on desktop windows isn't normal... does it take that long on any other mod when downloading a fresh copy of the source code?

**Reply by daisukidaisy** [reply]
I haven't tried any other mods idk

**Reply by Leather128** [reply]
if it happens on other mods, then it's your pc, if not then something is actually wrong, so could you try building another mod (or even the base game)?

**Reply by daisukidaisy** [reply]
i was just impatient about next version release idc i can wait if its happening as soon as it seems

**Reply by daisukidaisy** [reply]
i was just impatient about next version release idc i can wait if its happening as soon as it seems

**Reply by Leather128** [reply]
ok
